job-descriptions/software-engineer.md

```diff
@@ -42,12 +42,7 @@ Take a look at our public [near-term product roadmap](https://docs.sourcegraph.c
 
 1.  You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU)!
 1.  We schedule a 30 minute call to chat with you about Sourcegraph and to find out what you are looking for in your next role.
-1.  Choose your adventure!
-    - **Coding assignment**: We give you a well defined problem to solve and a 2 hour time limit. This is a lower time commitment than the paid project.
-    - **Paid project**: You propose working on one of the open issues in our open source repositories. Once we approve your proposal, you work on that issue until it is finished and merged. This is a higher time commitment than the coding assignment so we will pay you a flat rate of $250-$1000 depending on the size of the project after it is finished and merged.
-      - [Good first issues](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+org%3Asourcegraph+label%3A%22good+first+issue%22)
-      - [Extension requests](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+org%3Asourcegraph+label%3Aextension-request)
-      - Open a new issue to propose a feature or dev tool that you want!
+1.  We gather a code sample. We'll give you a well defined problem to solve and a 2 hour time limit.

```

Comment:
We got feedback that "code sample" sounded clinical (e.g. https://sourcegraph.slack.com/archives/C168VP0FM/p1550018256006800), so I would delete this bit.

```suggestion
1.  We give you a well defined problem to solve and a 2 hour time limit.
```